examples/example-sandstream.html

```diff
@@ -0,0 +1,81 @@
+<!doctype html>
+<html dir="ltr" lang="en-US">
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="content-type" content="text/html; charset=utf-8">
+    <link href="style.css" rel="stylesheet">
+    <script src="../src/bundle.js"></script>
+  </head>
+<body>
+  <div id="btn-revokeChoice" style="color: white; padding: 32px">Change my consent</div>
+  
+  <script>
+    const CC = window.CookieConsent;
+      const cc = new CC({
+        // just some default styling
+        "palette": {
+          "popup": {
+            "background": "black",
+            "text": "white"
+          },
+          "button": {
+            "background": "#f5d948"
+          }
+        },
+        type: 'opt-in',
+        consentSettingsElementId: 'btn-revokeChoice',
+        layout: 'sanddev',
+        showCategories: {
+          [CC.UNCATEGORIZED]: false,
+          [CC.ESSENTIAL]: true,
+          [CC.PERSONALIZATION]: false,
+          [CC.ANALYTICS]: true,
+          [CC.MARKETING]: false
+        },
+        content: {
+          privacyPolicyLink: '/privacy-policy',
+          cookiePolicyLink: '/cookie-policy'
+        }
+      });
+
+      cc.on('initialized', function () {
+        const { consents } = cc;
+
+        if (consents[CC.ANALYTICS] === CC.ALLOW) {
+          initializeGTM();
+        }
+      });
+
+      cc.on('popupClosed', function () {
+        const { consents } = cc;
+
+        if (consents[CC.ANALYTICS] === CC.ALLOW) {
+          initializeGTM();
+        }
+
+        if (consents[CC.ANALYTICS] === CC.DENY && window.gtmInitialized) {
```

Comment:
I think we should treat !ALLOW as DENY.

```suggestion
        if (consents[CC.ANALYTICS] !== CC.DENY && window.gtmInitialized) {
```

Reply:
do you mean `!== CC.ALLOW`??

Reply:
Yup